Assets/Scripts/BulletController.cs

```diff
@@ -1,26 +1,31 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class BulletController : MonoBehaviour
 {
-    public float speed = 10;
-
-    public float lifeTime = 5; //life of bullet in sec
-
+    private float lifeTime = 5f; // life of bullet in sec
```

Comment:
Why make them private? This makes it hard to make different types of bullet prefabs in the future.
Also its easier to dial in the perfect speed from the editor than constantly changing it from the code.